tests/cppunit/rwlock_test.cc

```diff
@@ -27,7 +27,7 @@
 TEST(LockManager, LockKey) {
   LockManager locks(8);
   std::vector<rocksdb::Slice> keys = {"abc", "123", "456", "abc", "123"};
-  for (const auto key : keys) {
```

Comment:
I think copying slice maybe more lightweight then using reference?

Reply:
Yeah, maybe passing by value is more effective. Don't know why there are many `const Slice&` even in rocksdb itself.